docs/source/markdown/podman-remote.1.md

```diff
@@ -39,6 +39,12 @@ Path to ssh identity file. If the identity file has been encrypted, Podman promp
 If no identity file is provided and no user is given, Podman defaults to the user running the podman command.
 Podman prompts for the login password on the remote server.
 
+Identity value resolution precedence:
+ - command line value
+ - environment variable `CONTAINER_SSHKEY`, if `CONTAINER_HOST` is found
+ - `containers.conf`
+ - `""`
```

Comment:
IDK, but maybe this
```suggestion
 - `""` (default value)
```

and if so, similar change for the URL below.

Reply:
I am just going to remove them, they don't really do anything.  I guess Podman remote will prompt if the user specifies nothing.